File: Modul-3/C/PRAK301-2310817320007-ZahraNabila.c
#include <stdio.h>

int main() 
{
    int a, b;

    printf("Masukkan Angka Acak : ");
    scanf("%d %d", &a, &b);

    if (a <= b) 
    { 
        printf("Angka Berurutan     : %d %d\n", a, b); 
    }
    else if (a >= b)
    { 
        printf("Angka Berurutan     : %d %d\n", b, a); 
    }
    return 0;
}

File: Modul-3/Python/PRAK301-2310817320007-ZahraNabila.py
a = int(input("Masukkan Angka Pertama   : "))
b = int(input("Masukkan Angka Kedua     : "))

if a <= b:
    pertama = a
    kedua   = b
else:
    pertama = b
    kedua   = a

print(f"Angka Berurutan          : {pertama} {kedua}")
